### PR TITLE
[Author] [New] Add scribbletune w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by walmik**
# Pull request template for new library

Pull request for issue: #12968
Related issue(s): # #12968

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/scribbletune/scribbletune
* Official website (optional, not the repository): https://scribbletune.com/
* NPM package url (optional): https://www.npmjs.com/package/scribbletune
* License and its reference: MIT
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 110
  * Count of stars: 2792
  * Count of forks: 186
* NPM download stats (optional):
  * Downloads in the last day: 
  * Downloads in the last week: 250+
  * Downloads in the last month: 1200+

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [ ] Auto-update setup
* [ ] Auto-update target/source is valid.
* [ ] Auto-update filemap is correct.

## Git commit checklist

* [ ] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [ ] The parent of the commit(s) in the PR is not older than 3 days.
* [ ] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by walmik** [comment]
@AilurusUmbra @Billy4195 @cy91244 @dakshshah96 @extend1994 @finch185277 @IonicaBizau @PeterDaveHello @petrgazarov @redox @ryankirkman @sufuf3 @thomasdavis

Hi folks, the build seems to be stuck for a few days. Could you kindly look into it and let me know if I need to do something that i may have missed?

**Comment by IonicaBizau** [comment]
It complains about the indent:


Try this (2 space indentation, and remove homepage):

```
{  
  "name":"scribbletune",
  "description":"Scribbletune is a JavaScript library that helps you construct musical ideas with JavaScript String & Array functions and export them as MIDI files.",
  "filename":"scribbletune.min.js",
  "author":{  
    "name":"Walmik",
    "email":"walmik.deshpande@gmail.com",
    "homepage":"https://scribbletune.com"
  },
  "license":"MIT",
  "repository":{  
    "type":"git",
    "url":"https://github.com/scribbletune/scribbletune.git"
  },
  "npmName":"scribbletune",
  "npmFileMap":[  
    {  
      "basePath":"dist",
      "files":[  
        "scribbletune.js"
      ]
    }
  ],
  "keywords":[  
    "music",
    "javascript",
    "webaudioapi",
    "tonejs",
    "scribbletune",
    "midi",
    "audio",
    "synth"
  ]
}
```

**Comment by walmik** [comment]
Wow thank you so much @IonicaBizau !!

**Comment by IonicaBizau** [comment]
I'm a musician too! I may use your library 🎶

**Comment by walmik** [comment]
Thats so good to know! I ve just added a new feature called `arp`, maybe you can tell if its a good addon thing or not.. it creates arpeggios out of chords and you can set the order of the arpeggio notes as well as set a custom pattern. This page has a sample wav at the very end if you want to see what it can generate) https://scribbletune.com/documentation/core/arp

**Comment by walmik** [comment]
HI @IonicaBizau do i need to do anything from my side for this PR to be merged... please let me know :)

**Comment by IonicaBizau** [comment]
I guess we are waiting for another confirmation (from another maintainer).

File: ajax/libs/scribbletune/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "scribbletune",
+  "description": "Scribbletune is a JavaScript library that helps you construct musical ideas with JavaScript String & Array functions and export them as MIDI files.",
+  "filename": "scribbletune.min.js",
+  "author": {
+    "name": "Walmik",
+    "email": "walmik.deshpande@gmail.com"
+  },
```
Comment (sufuf3) [inline ajax/libs/scribbletune/package.json]: Let's add the homepage, what do you think?
`"homepage": "https://scribbletune.com/"`
Many thanks

Comment (walmik) [reply ajax/libs/scribbletune/package.json]: Thank you for your comments @sufuf3 I ll add the home page info now.

Comment (walmik) [reply ajax/libs/scribbletune/package.json]: I ve added it now @sufuf3 :)

**Comment by walmik** [comment]
@AilurusUmbra @Billy4195 @cy91244 @dakshshah96 @extend1994 @finch185277 @IonicaBizau @PeterDaveHello @petrgazarov @redox @ryankirkman @sufuf3 @thomasdavis

Hi folks, is there anything more required for this PR to qualify for merging? Please help :)

**Comment by sufuf3** [comment]
Sorry that I am a little bit busy this week, I will look at this PR at the weekend.
Thank you

**Comment by walmik** [comment]
Hi @sufuf3 can you please take a look at this PR.

**Comment by walmik** [comment]
Hi @PeterDaveHello thank you for looking into this PR! Looking forward to getting it merged :)

**Comment by PeterDaveHello** [comment]
That file looks minified already but just not fully, I think we can remove the filename .min suffix and see if @walker can help make sure it'd fully minified, thanks.

**Pull request merged**